acceleration/TensorRT_inference_acceleration.ipynb

```diff
@@ -19,17 +19,15 @@
    ]
```

Comment:
<p>There is an extra space in the " In this part"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1347/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Hi @mingxin-zheng ,
     I think this is a visual bug of `ReviewNB`. When I opened the [notebook file](https://github.com/binliunls/tutorials/blob/update-TensorRT-acceleration-tutorial/acceleration/TensorRT_inference_acceleration.ipynb) directly in the github, the extra space didn't show again. 

Thanks,
Bin